data_model/src/expression.rs

```diff
@@ -858,6 +1072,23 @@ pub struct Where {
     pub values: btree_map::BTreeMap<ValueName, EvaluatesTo<Value>>,
 }
 
+impl core::fmt::Display for Where {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "`{} where: [", self.expression)?;
+
+        let mut first = true;
+        for (key, value) in &self.values {
+            if !first {
+                write!(f, ", ")?;
+            }
+            first = false;
+            write!(f, "`{}` = `{}`", key, value)?;
```

Comment:
```suggestion
            write!(f, "`{}` : `{}`", key, value)?;
```
This isn't assignment or equals. This is association.